### DISCUSSION TITLE
Maintenance status of this project

### DISCUSSION THREAD

**Discussion by thiphariel** [RandyKnapp/Auga#232] [General]
Hi there.

It's been the third time I'm back to Valheim on a custom dedicated modded server with a friend, and to be honest, we can't really play without Auga. It's such a great UI mod that the vanilla game should take ideas from.
But, it kind of feels abandonned on every patches that touches the UI. The mod breaks the game and usually doesn't get updated after a long while.

Can you be clear with an announcement or something on what is your current position about this project and how much effort you want to put on it? Maybe someone is willing to help or fork it to propose quicker patches fixes if you don't have time for that anymore. It's not a big deal to move on, maybe you aren't in Valheim modding anymore, not playing the game anymore etc.. It doesn't mean the project has to die with it ^^

You will probably tell me there are others UI mods out there, but none are even at Auga's ankle. I like this mod and hope it can live longer.

Thanks

**Comment by fouquetp** [comment]
Hello,

just back on Valheim to see this posted :(

Feel sad, Project Auga is such a good enhancement to the base game!

Anyway, thanks for the contribution :)
